app/code/Magento/InventoryImportExport/Model/Import/Command/CommandException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryImportExport\Model\Import\Command;
+
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Exception for command execution fails
+ */
+class CommandException extends LocalizedException
```

Comment:
If we introduce new Exception its name should be expressive enough. 

Currently `CommandException ` does not say a lot